content/applications/general/email_communication/email_template.rst

```diff
@@ -1,56 +1,128 @@
+is out of the scope of Odoo Support.
+
+.. seealso::
+   :doc:`../../../../services/support/what_can_i_expect`
+
+Defining a default reply to on the mail template
+================================================
+
+Although the *reply to* field is available within the mail templates, **this field is only used
+for mass mailing** mode (this means when sending templates on bulk emailing). Emails
+can be sent in bulk in almost every application that has a list view. Select the records where the
+emails are to be sent and click on the :guilabel:`Action` button. If there is the option to send an
+email, a mail composer with possible values to define will appear.
+
+.. image:: email_template/composer-mass-mailing.png
    :align: center
    :alt: Composer in mass mailing mode after selecting multiple quotations.
 
-You can also define them by default on the template:
+They can also be defined by default on the template:
 
 .. image:: email_template/reply-to-template-sales.png
    :align: center
    :alt: Reply-to field on template.
 
 Because of this, setting a value in this field is useless as the value defined will be totally
 ignored. The default *reply-to* value is the default catchall email address to ensure a
-communication between your customer and your Odoo database. For more information about the way
-the catchall works, please check :ref:`how to manage inbound messages <email_communication/inbound_messages>`.
+communication between the customer and the Odoo database. For more information about the way
+the catchall works, check :ref:`how to manage inbound messages
+<email_communication/inbound_messages>`.
 
 Transactional emails and corresponding URL for each company
 ===========================================================
 
 When using Odoo, multiple events trigger the sending of automated emails. These emails are known
-as transactional emails and sometimes contain links pointing to your Odoo database.
+as transactional emails and sometimes contain links pointing to the Odoo database.
 
 By default, links generated by the database use the dynamic web.base.url key defined in the system
 parameters. More information about this :ref:`parameter <domain-name/web-base-url>`.
 
 If the website application isn't installed, the web.base.url key will always be the default
 parameter used to generate all the links.
 
-It’s important to know that this key can only have a single value, meaning that in a
-multi-website/company database environment, even if you have a specific domain name for each
-website, the links generated to share a document or within a transactional email might remain the
+It's important to know that this key can only have a single value, meaning that in a
+multi-website/company database environment, even there is a specific domain name for each
+website, the links generated to share a document or within a transactional email may remain the
```

Comment:
I don't really understand what this is saying. I would rewrite it entirely.

Reply:
I've added an example: 

.. example::
   If the value of the web.base.url system parameter is equal to ``https://www.purpletest.com`` and
   there are two separate websites in Odoo with different URLS: ``https://www.purplecompany.com``
   and ``https://www.purpleteam.com``, the links created by Odoo to share a document or send a
   transactional email will come from the domain: ``https://www.purpletest.com``.